quarkchain/evm/messages.py

```diff
@@ -564,7 +564,11 @@ def _apply_msg(ext, msg, code):
 
     # Main loop
     if msg.code_address in ext.specials:
```

Comment:
Why not use

special_proc, enable_ts = ext.specials.get(msg.code_address, (None, 0))
if special_proc is not None and ext.block_timestamp >= enable_ts:
...

Reply:
good point. done